docs/tutorial.rst

```diff
@@ -113,54 +139,230 @@ The embddings can be loaded from disk using numpy:
+
+    # If the frame rate is provided, returns a list of timestamps
+    emb_list, ts_list = get_image_embedding(images, frame_rate=clip.fps, batch_size=32)
+    # ...otherwise, just the embeddings are returnedj
+    emb_list = get_image_embedding(images, batch_size=32)
+
+Multiple sequences of images can be provided as well
+
+.. code-block:: python
+
+    from moviepy.video.io.VideoFileClip import VideoFileClip
+
+    video_filepath1 = '/path/to/file1.mp4'
+    video_filepath2 = '/path/to/file2.mp4'
+    video_filepath3 = '/path/to/file2.mp4'
+
+    # Load video and get image frames
+    clip1 = VideoFileClip(video_filepath1)
+    images1 = np.array([frame for frame in clip1.iter_frames()])
+    clip2 = VideoFileClip(video_filepath2)
+    images2 = np.array([frame for frame in clip2.iter_frames()])
+    clip3 = VideoFileClip(video_filepath3)
+    images3 = np.array([frame for frame in clip3.iter_frames()])
+
+    image_list = [images1, images2, images3]
+    frame_rate_list = [clip1.fps, clip2.fps, clip3.fps]
+
+    # If the frame rates is provided...
+    emb_list, ts_list = get_image_embedding(image_list, frame_rate=frame_rate_list, batch_size=32)
+    # or if a single frame rate applying to all sequences is provided, returns a list of timestamps
+    emb_list, ts_list = get_image_embedding(image_list, frame_rate=clip1.fps, batch_size=32)
+    # ...otherwise, just the embeddings are returnedj
+    emb_list = get_image_embedding(image_list, batch_size=32)
+
+
+
+
+Image files can can also be processed with just the filepath:
+
+.. code-block:: python
+
+    import openl3
+    import numpy as np
+
+    image_filepath = '/path/to/file.png'
+
+    # Save the file to '/path/to/file.npz'
+    openl3.process_image_file(image_filepath)
+
+    # Preload model
+    model = openl3.models.load_image_embedding_model(input_repr="mel256", content_type="music",
+                                                     embedding_size=512)
+    openl3.process_image_file(image_filepath, model=model)
+
+    # Save the file to `/path/to/file_suffix.npz`
+    openl3.process_image_file(image_filepath, model=model, suffix='suffix')
+
+    # Save the file to '/different/dir/file_suffix.npz'
+    openl3.process_image_file(image_filepath, model=model, suffix='suffix', output_dir='/different/dir')
+
+Multiple files can be processed as well (with a specified batch size used for processing) as follows:
+
+.. code-block:: python
+
+    import openl3
+    import numpy as np
+
+    image_filepath1 = '/path/to/file1.png'
+    image_filepath2 = '/path/to/file2.png'
+    image_filepath3 = '/path/to/file3.png'
+    image_filepath_list = [image_filepath1, image_filepath2, image_filepath3]
+
+    # Saves embeddings to '/path/to/file1.npz', '/path/to/file2.npz', and '/path/to/file3.npz'
+    openl3.process_image_file(image_filepath_list, batch_size=32)
+
+
+Processing video files
+**********************
+
+Video files can also be processed to extract both audio and image embeddings:
```

Comment:
Will the timestamps of the audio and image embeddings be the same? Aligned? We need to provide some info here.